modules/meb_api/lib/dgi/automation/service.rb

```diff
@@ -27,9 +31,7 @@ def end_point
         end
 
         def json
-          # Passes Back User SSN in the Re body
-          # { "ssn": '539139735' }
-
+          # { "ssn": "539139735" }
```

Comment:
Instead of passing this param in the controller ideally we would pass in the user's ssn dynamically from this service class. Any feedback on how we might be able to accomplish is welcomed.